### DISCUSSION TITLE
Wheels installed into a subdirectory

### DISCUSSION THREAD

**Discussion by mikegrogan** [wheels-dev/wheels#1430] [Ideas]
Wheels is setup to be installed into the root directory. I install it into subfolders, such as /project1, /project2. I have a few different wheels projects. Maybe I could merge them at this point, but for now they have different logins and focuses, so I kept them separate when I set them up years ago. I just finally upgrade to 2.5.1 from 1.4 and wanted to document some of the tweaks I've had to make to the core code. Wondering if there's an easier way to tell wheels it lives outside of the root. And maybe some of this won't apply to version 3.0. I think I already saw a fix in there for one of the issues.

Changes I've made:
* On initial load of the wheels info, routes, etc. internal pages none of the CSS loads. I changed https://github.com/cfwheels/cfwheels/blob/v2.5.1/wheels/public/layout/_header_simple.cfm and a few other similar pages in that directory to add the webpath before the wheels folder, such as <script src="#$get('webPath')#wheels/public/assets/js/jquery.min.js"></script>
*  My application name is the same as my folder name (under the root). I updated https://github.com/cfwheels/cfwheels/blob/v2.5.1/wheels/events/onapplicationstart.cfm#L970 to change the local.url to the correct path after a reload. local.url = "/#this.name##cgi.path_info#";. Otherwise it'll go to the root of our site which is outside of cfwheels and gets confusing.
* This isn't an issue, but just to be complete, I also change the URL redirect rules to include my sub folder too, so the url rewriting actually works.

Wondering if anyone else has done something more permanent or similar to this?

**Comment by chapmandu** [comment]
I did this a while ago.. it may or may not help. 

https://github.com/chapmandu/UpgradeAdvisor
